gravitee-am-service/src/main/java/io/gravitee/am/service/impl/CertificateServiceImpl.java

```diff
@@ -232,10 +232,10 @@ public Single<Certificate> create(String domain, NewCertificate newCertificate,
                 })
                 .map(this::getValid)
                 .flatMap(certificate -> certificateRepository.create(certificate))
-                // create event for sync process
-                .flatMap(certificate -> {
+                .doOnSuccess(certificate -> {
```

Comment:
question: do we really want to fire&forget the event creation ? 🤔 If an error occures the event may not be persist in DB and so the GW will never see the change. Before this update, an error will be display on the UI, after this change, the UI will display a successful response but the GWs will not see the updates...